brayns/Brayns.cpp

```diff
@@ -633,26 +608,18 @@ struct Brayns::Impl : public PluginAPI
 
     void _increaseAnimationFrame()
```

Comment:
Move to AnimationParams::jumpFrames(int frames)

Reply:
done